src/Controller/NphOrderLookupController.php

```diff
@@ -54,13 +81,11 @@ public function index(
             }
             $this->addFlash('error', $crossSiteErrorMessage ?? 'Order ID not found');
```

Comment:
Also, cross site check doesn't apply for biobank users so you might need to extract that logic out.